crates/sargon/src/gateway_api/models/types/request/state/entity/request.rs

```diff
@@ -1,36 +0,0 @@
-use crate::prelude::*;
-
-#[derive(
-    Clone,
-    Debug,
-    PartialEq,
-    Eq,
-    Serialize,
-    Deserialize, /* Deserialize so we can test roundtrip of JSON vectors */
-)]
-pub struct StateEntityDetailsRequest {
```

Comment:
moved into `state/entity/details/request` to follow folder structure with endpoint path.